### PR TITLE
Update doc to reflect new tracking descriptions

### PR DESCRIPTION
**Pull request by Brooks-morrison**
https://pokitdok.atlassian.net/browse/ENTRY-4436

### CODE REVIEW

File: source/includes/_activities.md [hunk 1]
```diff
@@ -354,4 +354,45 @@ Throughout processing, activities may transition through the following states:
...
+<a name="tracking-description"></a>
+
+
+Tracking descriptions summarize the tracking/outcome of claims:
+
+<!--- beginning of table -->
+
+|State             | Description |
+|:------------------|:--------------------------------------------------------------------------------------
+|submitting        |Claim is in the process of beeing submitted to payer |
```
Comment (corbinbs) [inline source/includes/_activities.md]: `beeing` -> `being`
🐝

File: source/includes/_activities.md [hunk 2]
```diff
@@ -354,4 +354,45 @@ Throughout processing, activities may transition through the following states:
...
+|claim_not_found  | Claim cannot be found in payer's adjudication system |
+|acknowledged_split| Claim has been split upon acceptance into adjudication system  |
+|claim_pended     | No remittance has been issued, or only part of the claim has been paid  |
+|pending_adjudication |Claim is in payer's system and is pending adjudication  |
+|pending_additional_information |Claim is waiting for additional information from submitter  |
+|finalized        |The claim cycle has been completed and no additional action will be taken  |
+|finalized_revised|Adjudication info has been changed |
+|finalized forwarded|Claim processing is complete. Claim has been forwarded to a different entity  |
+|adjudicated_not_paid|No payment is forthcoming |
+|additional_information_ requested|Additional information has been requested by the payer  |
```
Comment (corbinbs) [inline source/includes/_activities.md]: looks like a ` ` in `additional_information_ requested`

File: source/includes/_activities.md [hunk 3]
```diff
@@ -354,4 +354,45 @@ Throughout processing, activities may transition through the following states:
...
+|processing_error |Error in the payer's system |
+|response_received|Default tracking description, check result for more detailed information|
```
Comment (corbinbs) [inline source/includes/_activities.md]: might adjust this to be `Default tracking description once a response has been received for a claim,`...

File: source/includes/_activities.md [hunk 4]
```diff
@@ -354,4 +354,45 @@ Throughout processing, activities may transition through the following states:
...
 
+### Tracking Description Table
+<a name="tracking-description"></a>
+
+
+Tracking descriptions summarize the tracking/outcome of claims:
+
+<!--- beginning of table -->
+
+|State             | Description |
```
Comment (corbinbs) [inline source/includes/_activities.md]: should `State` be `Tracking Description Value` or something similar?

Comment (Brooks-morrison) [reply source/includes/_activities.md]: yep

**Pull request merged**